### DISCUSSION TITLE
EGU 2023 informal and splinter meeting

### DISCUSSION THREAD

**Discussion by malininae** [ESMValGroup/Community#90] [General]
Hi @ESMValGroup/esmvaltool-developmentteam @ESMValGroup/esmvaltool-coreteam @ESMValGroup/userengagementteam , 

I was wondering, if someone wants to have an informal meeting at EGU? I was thinking about grabbing a dinner or having beers one evening. I started using the tool during the pandemic, so would be so good to meet in person!

**Comment by bouweandela** [comment]
Quite a few ESMValTool developers will be present at EGU this year. I just requested a splinter meeting for the ESMValTool community, so if that is granted that could be an opportunity to meet up as well.

**Reply by malininae** [reply]
@bouweandela sounds good, keep us posted please! I'm so excited to meet everyone!

**Reply by bouweandela** [reply]
Here is the link to the splinter meeting: https://meetingorganizer.copernicus.org/EGU23/session/47852
